requirements.txt

```diff
@@ -1,4 +1,4 @@
-ansible-core>=2.16,<2.17.5
+ansible-core>=2.17.5,<2.18.1
```

Comment:
@haidaraM  That is not what's written in the commit message!?
~~~
[chore(deps): update ansible-core requirement from <2.17.5,>=2.16 to >=2.16,<2.18.1]
~~~
??
To me it was surprising that a patch release drops support for the environment I am currently using.

Reply:
Hmmm! Seems like I forgot to update the commit/PR title, my bad. 

Had some issues with the versions when working on this PR. Which environment/Ansible version are you refering to? Will fix it